marlowe-playground-client/static/css/typo.css

```diff
@@ -76,4 +76,8 @@ h6 {
 
 .uppercase {
   text-transform: uppercase;
-}
\ No newline at end of file
+}
+
+.underline {
+  text-decoration: underline;
+}
```

Comment:
I believe Bootstrap will do this for you if you wrap the thing in `u` tag.

Wait, you're using `.underline` in `a` tags. Weird that they're not already underlined? Normally you'd keep the default (because accessibility) and use something like https://getbootstrap.com/docs/4.4/utilities/text/#text-decoration when you need to remove it.

Reply:
not using bootstrap. underlining of `a` disabled globally.